src/web/releases.rs

```diff
@@ -579,6 +580,13 @@ pub(crate) async fn search_handler(
             return Err(AxumNope::NoResults);
         }
 
+        static RE: Lazy<Regex> = Lazy::new(|| Regex::new(r"[?&]sort=([^&]+)").unwrap());
```

Comment:
I was wondering for a second why we need to re-parse here, but then I realized that pagination is correctly working, including the sorting, only the dropdown box shows the wrong value on the further pages. This is probably worth a comment. 

I don't like using regex here, we already have `form_urlencoded` in our dependencies, and could just use `form_urlencoded::parse` here.

Reply:
Nice suggestion, I will try this package.